mordred/task_panels.py

```diff
@@ -39,18 +39,6 @@
 # Header mandatory in Ellasticsearc 6 (optional in earlier versions)
 ES6_HEADER = {"Content-Type": "application/json"}
 
-# We don't have this data so it just works for this value
-ES6_KIBANA_INIT_URL = "http://kibiter:5601"
-ES6_KIBANA_INIT_URL += "/api/kibana/settings/indexPattern:placeholder"
-ES6_KIBANA_INIT_DATA = '{"value": "*"}'
-# We need the version before the .kibana exists so we can not find it
-ES6_KIBANA_VERSION = "6.1.0-1"
-ES6_KIBANA_INIT_HEADERS = {
```

Comment:
I would rather keep this constant, to easy the code below (and spare the variable k6_init_headers, and its assignment on the fly).

Reply:
So the only global constant is then `ES6_KIBANA_INIT_DATA` because `ES6_KIBANA_VERSION` is now defined in mordred config. I have left then:

```
ES6_KIBANA_INIT_DATA = '{"value": "*"}'
ES6_KIBANA_INIT_URL_PATH = "/api/kibana/settings/indexPattern:placeholder"
```

Reply:
I was referring mainly to `ES_KIBANA_INIT_HEADERS`, sorry for not being clear enough.

Reply:
You can not define this as a global constant because it depends of the version that is a variable defined in mordred config (`self.conf['panels']['kibiter_version']`), and it is not available when this global variable will be defined. 

The headers could be defined partially globally:

```
ES_KIBITER_INIT_HEADERS = {
                    "Accept": "application/json",
                    "Content-Type": "application/json",
                    "kbn-version": None
}
```

and later, complete it with the kbn-version value but it does not make sense IMHO (I found it too complex).

Reply:
I was meaning something like you say, but I agree it is a bit weird. So, if you prefer, let's keep it as it is now.